src/main/java/com/bloomtechlabs/fp/entities/EmploymentHistory.java

```diff
@@ -0,0 +1,61 @@
+package com.bloomtechlabs.fp.entities;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name ="employment_histories")
+public class EmploymentHistory {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "client_id")
+    private Long clientId;
+
+    @Column(name = "currently_employed")
+    private Boolean currentlyEmployed;
+
+    @Column(name = "skill_certifications")
+    private String skillCertifications;
+
+    public EmploymentHistory() {
+    }
+
+    public EmploymentHistory(Long clientId, Boolean currentlyEmployed, String skillCertifications) {
```

Comment:
Do we want to include an input parameter for clientId if the table is auto generating the value?

Reply:
yes, that is the clientId that we got from the clients table in the schema. the id that was auto-generated in this entity was not included.